psi-ms.obo

```diff
@@ -20557,6 +20558,13 @@ name: quality control data format
 def: "Grouping term for quality control data formats." [PSI:MS]
 is_a: MS:1001459 ! file format
 
+[Term]
+id: MS:1003162
+name: PTX-QC
+def: "Proteomics (PTX) - QualityControl (QC) software for QC report generation and visualization" [DOI:10.1021/acs.jproteome.5b00780, PMID:26653327, https://github.com/cbielow/PTXQC/]
+is_a: MS:1001456 ! analysis software
```

Comment:
Should this be a new sub-type of `software` "quality control software" instead of "analysis software"?

Reply:
its kind of both, so I'm happy either way

Reply:
If we don't already have a QC software category (and I'm not convinced we need it), analysis software seems fine. QC software would be a child of analysis software IMO.